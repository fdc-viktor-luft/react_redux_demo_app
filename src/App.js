// @flow
import React from 'react';
import { PageHeader } from 'react-bootstrap';

import { ListComponent } from './components/ListComponent';
import { SumComponent } from './components/SumComponent';
import { RequestJokeComponent } from './components/RequestJokeComponent';

export class App extends React.Component {

    render() {
        return (
            <div>
                <PageHeader>React + Redux Demo</PageHeader>
                <ListComponent />
                <SumComponent />
                <RequestJokeComponent />
            </div>
        );
    }
}

