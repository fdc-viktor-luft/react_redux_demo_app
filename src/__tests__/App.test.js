import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { App } from '../App';

describe('App', () => {
    it('[snapshot] it renders correctly', () => {
        const app = shallow(<App />);
        expect(toJson(app)).toMatchSnapshot();
    });
});
