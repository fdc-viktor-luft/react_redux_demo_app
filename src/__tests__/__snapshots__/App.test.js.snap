exports[`App [snapshot] it renders correctly 1`] = `
<div>
  <PageHeader
    bsClass="page-header">
    React + Redux Demo
  </PageHeader>
  <Connect(DisconnectedListComponent) />
  <Connect(DisconnectedSumComponent) />
  <Connect(DisconnectedRequestJokeComponent) />
</div>
`;
